NGitLab/Models/CommitCreate.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Text.Json.Serialization;
 
 namespace NGitLab.Models
 {
     public class CommitCreate
     {
-        [Required]
-        [JsonPropertyName("id")]
+        [Obsolete("'Project Id' is ignored in the POSTed JSON; it is actually specified through the endpoint URL.")]
```

Comment:
Can you add JsonIgnore on them too? To avoid sending 'projectId' or 'ProjectId' property